graph/minimum_diameter_spanning_tree/task.md

```diff
@@ -0,0 +1,69 @@
+## @{keyword.statement}
+
+@{lang.en}
+
+Given a weighted undirected graph with $N$ vertices and $M$ edges. The $i$-th edge is $(a_i, b_i)$ and has a weight of $c_i$. This graph may not be simple.
+Find the minimum diameter spanning tree.
```

Comment:
```suggestion
Find a minimum diameter spanning tree.
```
I think "a minimum spanning tree" is better than "the minimum spanning tree" because it may not be unique.